client/state/reader/posts/normalization-rules.js

```diff
@@ -84,25 +83,6 @@ export function classifyPost( post ) {
 		displayType ^= DISPLAY_TYPES.PHOTO_ONLY;
 	}
 
-	if ( canonicalImage ) {
```

Comment:
The display types `LARGE_BANNER`, `LANDSCAPE_BANNER` and `PORTRAIT_BANNER` are no longer used. We don't need to assign `THUMBNAIL` specifically, because all unclassified cards with one usable image will get that layout by default.